### DISCUSSION TITLE
Whatsapp is out of date. Updating now...

### DISCUSSION THREAD

**Discussion by lcompare** [devlikeapro/waha#401] [General]
Hello, I am getting this message since Saturday. Is there any solution for it? I see there is no new image yet.

![Screenshot from 2024-07-01 11-22-21](https://github.com/devlikeapro/waha/assets/3417971/32c207d0-169d-4312-866e-e002b9db79aa)

**Comment by devlikepro** [comment]
Hi! Make sure you're using the latest version - `devlikeapro/waha-plus` (or `waha` docker image), it's been renamed
- https://boosty.to/wa-http-api/posts/6a58df1a-6e47-4ed2-a2b0-b673e99f2070
- https://boosty.to/wa-http-api/posts/a6c01a2d-704e-4066-978a-890e84de9da1

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by lcompare** [reply]
That was it. Thank you! Didn't know it was renamed.
